### DISCUSSION TITLE
Make error when execute ./make_devel.sh

### DISCUSSION THREAD

**Discussion by Million-Miles** [madMAx43v3r/chia-plotter#780] [General]
![image](https://user-images.githubusercontent.com/1585846/125039837-85a28100-e0c9-11eb-8546-8d0e0f12922a.png)

My system is ubuntu, I just follow the steps to build, but when I execute ./make_devel.sh， I encounter an error as below show, please help to address the issue, thanks a lot!

chia@chia:~/Downloads/chia-plotter$ ./make_devel.sh
-- Architecture: x86_64
CMake Warning (dev) at lib/bls-signatures/CMakeLists.txt:11 (set):
  implicitly converting 'INTEGER' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at lib/bls-signatures/CMakeLists.txt:12 (set):
  implicitly converting 'INTEGER' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at lib/bls-signatures/CMakeLists.txt:13 (set):
  implicitly converting 'INTEGER' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Build python bindings: false
-- Build tests: false
-- Build benchmarks: false
-- Found libsodium
-- Sodium include dir = /usr/include
GMP_INCLUDES=GMP_INCLUDES-NOTFOUND
-- Could NOT find gmp (missing: GMP_INCLUDES GMP_LIBRARIES GMP_VERSION_OK)
CMake Warning (dev) at lib/bls-signatures/CMakeLists.txt:54 (set):
  implicitly converting 'INTEGER' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at lib/bls-signatures/CMakeLists.txt:65 (set):
  implicitly converting 'INTEGER' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at lib/bls-signatures/CMakeLists.txt:86 (set):
  implicitly converting 'INTEGER' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at lib/bls-signatures/CMakeLists.txt:87 (set):
  implicitly converting 'INTEGER' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at lib/bls-signatures/CMakeLists.txt:88 (set):
  implicitly converting 'INTEGER' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Relic will be built from: origin/main
[ 11%] Performing update step for 'relic-populate'
fatal: 无法访问 'https://github.com/relic-toolkit/relic.git/'：GnuTLS recv error (-110): The TLS connection was non-properly terminated.
CMake Error at /home/chia/Downloads/chia-plotter/build/_deps/relic-subbuild/relic-populate-prefix/tmp/relic-populate-gitupdate.cmake:59 (message):
  Failed to fetch repository 'https://github.com/relic-toolkit/relic.git'

gmake[2]: *** [CMakeFiles/relic-populate.dir/build.make:116：relic-populate-prefix/src/relic-populate-stamp/relic-populate-update] 错误 1
gmake[1]: *** [CMakeFiles/Makefile2:95：CMakeFiles/relic-populate.dir/all] 错误 2
gmake: *** [Makefile:103：all] 错误 2

CMake Error at /usr/share/cmake-3.18/Modules/FetchContent.cmake:987 (message):
  Build step for relic failed: 2
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FetchContent.cmake:1082:EVAL:2 (__FetchContent_directPopulate)
  /usr/share/cmake-3.18/Modules/FetchContent.cmake:1082 (cmake_language)
  /usr/share/cmake-3.18/Modules/FetchContent.cmake:1125 (FetchContent_Populate)
  lib/bls-signatures/src/CMakeLists.txt:20 (FetchContent_MakeAvailable)

-- Configuring incomplete, errors occurred!
See also "/home/chia/Downloads/chia-plotter/build/CMakeFiles/CMakeOutput.log".
See also "/home/chia/Downloads/chia-plotter/build/CMakeFiles/CMakeError.log".
make: *** 没有指明目标并且找不到 makefile。 停止。

**Comment by Million-Miles** [comment]
OK, Through VPN connection, the issue didn't occur.

**Reply by StoneCX** [reply]
same error in china ,haha
